scarb/src/core/publishing/manifest_normalization.rs

```diff
@@ -73,6 +73,7 @@ fn generate_package(pkg: &Package) -> Box<TomlPackage> {
             .clone()
             .map(|_| MaybeWorkspace::Defined((Utf8PathBuf::from(DEFAULT_README_FILE_NAME)).into())),
         repository: metadata.repository.clone().map(MaybeWorkspace::Defined),
+        include: None,
```

Comment:
Why is it empty?

Shouldn’t packaging with the normalized manifest produce the same result as packaging with the original one, including the same additional files?


It also is normalized in Cargo: https://github.com/rust-lang/cargo/blob/351b421bed006071ecd7f79cc81335d879f2ef45/src/cargo/util/toml/mod.rs#L1558